javascript/uuidgenerator.js

```diff
@@ -0,0 +1,29 @@
+/* eslint no-console: "error" */
+
+// npm install uuid
+
+/* ----------------------------------- */
+/* ----------------------------------- */
+
+// using version 4 to create uuid
+// generates a random uuid
+
+const { v4: uuidv4 } = require("uuid");
+Console.log(`${uuidv4()}`);
+
+// output
+// da8f5839-63f0-4c25-a7b1-a0172a03904b
+
+/* ----------------------------------- */
+/* ----------------------------------- */
+
+// here we use only v4,
+// to use v5 uncomment it using ctrl + /
+// using version 5 to create uuid
+// generates a random uuid
+
+const { v5: uuidv5 } = require("uuid");
+Console.log(`${uuidv5()}`);
```

Comment:
Codacy found an issue: ['Console' is not defined.](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6281261)